.github/workflows/main.yml

```diff
@@ -101,4 +79,4 @@ jobs:
         with:
           app-name: ${{ env.AZURE_WEBAPP_NAME }}
           publish-profile: ${{ secrets.AZURE_WEBAPP_PUBLISH_PROFILE }}
-          images: "${{ env.IMAGE_NAME }}:main"
+          images: "${{ env.IMAGE_NAME }}:${{ github.sha }}"
```

Comment:
can you please check an alternative way to redeploy the latest version of a tag? Why is it not happening for the frontend?